tests/integration_test.rs

```diff
@@ -50,18 +50,16 @@ mod tests {
         let mut mm = get_momento_instance();
         let result = mm.create_cache(cache_name).await.unwrap_err();
         let _err_msg = "Cache name cannot be empty".to_string();
-        assert!(matches!(
-            result,
-            MomentoError::InvalidArgument(_err_message)
-        ))
+        assert!(matches!(result.to_string(), _err_message))
```

Comment:
```suggestion
        assert_eq!(result.to_string(), "Cache name cannot be empty");
```
If the types are comparable then `assert_eq!` is better than doing `assert!(matches!(...))` because it will print out the values when it fails.

It looks like there's the same thing going on in a number of different test cases.

Reply:
yeah, I agree - not sure why that was done that way. I don't want to conflate fixing that with this change though